TeamSnapSDK/SDK/CollectionJSON/TSDKCollectionObject.m

```diff
@@ -31,8 +31,8 @@ @implementation TSDKCollectionObject
 static NSMutableDictionary *_queryDictionary;
 static NSMutableDictionary *_classURLs;
 
-+ (void)initialize
-{
++ (void)initialize {
+    [super initialize];
```

Comment:
A call to `super` is not required for `initialize`. This method will be called once for every subclass of `TSDKCollectionObject` when that subclass is initialized by the runtime.

Reply:
I'll yank it. I am going to add another test as well.

Reply:
Removed.